src/chi/components/picker/picker.scss

```diff
@@ -0,0 +1,156 @@
+@import 'variables';
+@import 'mixins';
+
+$border: 0.0625rem;
+$sizes: (
+  sm: (
+    font-size: $text-small,
+    font-weight: 400,
+    line-height: 1rem,
+    padding: (0.5rem - $border) (1rem - $border) (0.5rem - $border) (1.5rem - $border)
+  ),
+  md: (
+    font-size: $text,
+    font-weight: 400,
+    line-height: 1rem,
+    padding: (0.75rem - $border) (1rem - $border) (0.75rem - $border) (1.5rem - $border)
+  ),
+  lg: (
+    font-size: $text-large,
+    font-weight: 600,
+    line-height: 1rem,
+    padding: (1rem - $border) (1.5rem - $border) (1rem - $border) (1.5rem - $border)
+  )
+);
+
+.chi {
+  .m-picker {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+
+    input {
+      &[type='checkbox'],
+      &[type='radio'] {
+        &.a-picker {
+          left: -9999px;
+          position: absolute;
+
+          & + label {
+            align-items: center;
+            background-color: $text-white;
+            border: solid 0.0625rem set-color(grey, 30);
+            border-radius: 0.1875rem;
+            box-shadow: map-get($shadow, 1);
+            color: $dark-color;
+            display: flex;
+            font-size: $text;
+            justify-content: space-between;
+            padding: 1rem - $border;
+            margin-bottom: 0.5rem;
+            transition: all 0.2s;
+            width: 100%;
+
+            & div {
+              text-transform: none;
+
+              & :not(label) {
+                display: flex;
+                flex-direction: column;
+                line-height: 1rem;
+              }
+
+              &:last-of-type {
+                text-align: right;
+              }
+            }
+
+            &:hover {
+              background-color: set-color(grey, 10);
+              cursor: pointer;
+            }
+          }
+
+          &:checked {
+            & + label {
+              border: solid 0.0625rem $brand-color;
+              box-shadow: inset 0 0 0 0.0625rem $brand-color;
+              color: $brand-color;
+              font-weight: 600;
+
+              & div {
+                font-weight: 600;
+              }
+
+              & span {
+                color: $dark-color;
+              }
+
+              & .a-input {
+                & + label {
+                  &::before {
+                    background-color: $brand-color;
```

Comment:
This background-color is not right. It's applying :checked checkbox styles to :checked radio buttons.